### PR TITLE
small updates to Colab demo

### PR DESCRIPTION
**Pull request by MMathisLab**
- file naming consistency
- file path; otherwise it would not run for me: 
Screenshot 2024-05-15 at 08 52 37
-remove unnecessary descriptions; as it takes up horizontal space in Colab and reduces the visibility of the code.

### CODE REVIEW

**Comment by C-Achard** [comment]
Thanks @MMathisLab ! I will have a look and merge soon.

File: notebooks/Colab_inference_demo.ipynb [hunk 1]
```diff
@@ -153,12 +177,12 @@
...
-      "metadata": {},
+      "metadata": {
+        "id": "O0jLRpARAm-0"
+      },
       "outputs": [],
       "source": [
-        "# @title Load demo image and inference configuration\n",
-        "#@markdown This cell loads a demo image and load the inference configuration.\n",
-        "demo_image_path = \"./CellSeg3D/examples/c5image.tif\n",
+        "demo_image_path = \"/content/CellSeg3D/examples/c5image.tif\"\n",
```
Comment (C-Achard) [inline notebooks/Colab_inference_demo.ipynb:185]: I'll check again before merging

File: notebooks/Colab_inference_demo.ipynb [hunk 2]
```diff
@@ -3,36 +3,42 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "colab_type": "text",
-        "id": "view-in-github"
+        "id": "view-in-github",
+        "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/AdaptiveMotorControlLab/CellSeg3d/blob/main/notebooks/colab_inference_demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/MMathisLab/CellSeg3d/blob/main/notebooks/Colab_inference_demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
[the remark this answers is not in the document]
Comment (MMathisLab) [reply notebooks/Colab_inference_demo.ipynb:10]: def. can change, agree

**Review by C-Achard — APPROVED** [review]

**Comment by C-Achard** [comment]
All good, merging now, thanks again @MMathisLab (and sorry for the delay)

**Pull request merged**